frontend/interfaces/user.ts

```diff
@@ -4,6 +4,7 @@ import teamInterface, { ITeam } from "./team";
 export default PropTypes.shape({
   email: PropTypes.string,
   force_password_reset: PropTypes.bool,
+  api_only: PropTypes.bool,
```

Comment:
It looks like changes related to API-only user made it's way into this PR. I'm not sure if this matters.

Reply:
I put it in there because there was a build error without it. Better to get that merged in now than wait for my redux store state stuff to be sorted for these changes to be merged!

Reply:
Love it. Ship it. :shipit: